Cargo.toml

```diff
@@ -43,13 +43,16 @@ path = "src/bin/noir_cli/main.rs"
 required-features = ["bins"]
 
 [features]
+default = ["std", "bins"]
 std = []
 bins = [
     "dep:clap",
     "dep:hex",
     "dep:anyhow",
-    "dep:thiserror",
     "dep:regex",
-    "dep:serde_json",
-    "dep:serde",
+    "dep:log",
+    "dep:env_logger"
 ]
+
+[package.metadata.cargo-udeps.ignore]
+development = ["assert_cmd", "assert_fs", "predicates"]
```

Comment:
If you do the changes in `Makefile.toml` you can remove it

Reply:
Done in https://github.com/zkVerify/ultraplonk_verifier/pull/5/commits/35711515b71a0e3f12517b6cd9005fe5cdb342a9, thank you!